userspace/libsinsp/sinsp.cpp

```diff
@@ -1846,6 +1846,11 @@ const scap_machine_info* sinsp::get_machine_info()
 	return m_machine_info;
 }
 
+const scap_agent_info* sinsp::get_agent_info()
+{
+	return m_agent_info;
```

Comment:
right now we are not populating `m_agent_info` is this correct?

Reply:
+1